R/mmrm.R

```diff
@@ -123,33 +115,30 @@ as_mmrm_formula <- function(mmrm_df, cov_struct) {
 }
 
 
-#' Extract glmmTMB model parameters
+#' Extract parameters from a MMRM model
 #'
 #' Extracts the beta and sigma coefficients from an MMRM model created
-#' by [glmmTMB::glmmTMB()].
-#' Also returns theta for use in providing initial values to subsequent calls.
+#' by [mmrm::mmrm()].
 #'
-#' @param fit an object created by [glmmTMB::glmmTMB()]
-#' @importFrom glmmTMB fixef VarCorr getME
+#' @param fit an object created by [mmrm::mmrm()]
 #'
 extract_params <- function(fit) {
-    beta <- fixef(fit)$cond
+    beta <- coef(fit)
     names(beta) <- NULL
 
-    sigma <- VarCorr(fit)$cond
-    sigma <- lapply(sigma, function(x) {
-        x <- as.matrix(as.data.frame(x))
-        colnames(x) <- NULL
-        rownames(x) <- NULL
-        return(x)
+    sigma <- fit$cov
```

Comment:
```suggestion
    sigma <- VarCorr(fit)
```
should work as well here